pages/blog/2023-summary.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "Annual Review: State of AsyncAPI Initiative 2023"
+date: 2024-01-18T06:00:00+01:00
+type: Communication
+tags:
+  - Project Status
+cover: /img/posts/2023-summary/blog-banner-2023.webp
+authors:
+  - name: Thulisile Sibanda
+    photo: /img/avatars/thulieblack.webp
+    link: https://www.linkedin.com/in/v-thulisile-sibanda/
+    byline: AsyncAPI Community Manager
+excerpt: '2023 Year in Review'
+featured: true
+---
+
+It's almost unbelievable that we are in 2024. And if you are reading this, Happy New Year to you and your loved ones. Continuing the tradition, we look at how the community has grown over the last 365 days and look forward to our next phase.
+
+If you are curious about our past achievements, [please read the previous summary](https://www.asyncapi.com/blog/2022-summary).
+
+## Community Metrics
+What better way to reflect on the past year than with numbers? This is the most significant way to gauge our growth and measure our visible impact as an open-source community.
```

Comment:
```suggestion
What better way to reflect on the past year than with numbers? Data is the most significant way to gauge our growth and measure our visible impact as an open-source community.
```

An unclear antecedent (i.e., `this`) occurs when a pronoun starts a sentence and refers back to a vague idea rather than to an actual noun or explicit statement.